docs/content/docs/deployment/filesystems/gcs.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Google cloud storage
+weight: 3
+type: docs
+aliases:
+  - /deployment/filesystems/gcs.html
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Google Cloud Storage
+
+[Google Cloud storage](https://cloud.google.com/storage) (GCS) provides cloud storage for variety of usecases. You can use it for **reading** and **writing data**.
+
+
+
+You can use GCS objects like regular files by specifying paths in the following format:
+
+```plain
+gs://<your-bucket>/<endpoint>
+```
+
+The endpoint can either be a single file or a directory, for example:
+
+```java
+// Read from GSC bucket
+env.readTextFile("gs://<bucket>/<endpoint>");
+
+// Write to GCS bucket
+stream.writeAsText("gs://<bucket>/<endpoint>");
+
+```
+
+### Libraries
+
+You would require to include the following libraries to `/lib` for connecting flink with gcs. 
```

Comment:
Can the GCS filesystem be used as a Flink plugin, or is it really the case that it only works by putting the jar files into the lib directory (and I assume you meant Flink's lib directory, rather than `/lib`).

```suggestion
You must include the following libraries in Flink's `lib` directory to connect Flink with gcs:
```

Reply:
Hi, I tried running the same application via copying jar's to `plugins/gcs`. It does not seem to work with that.